### PR TITLE
Clean up code, take #2

### PR DESCRIPTION
**Pull request by bolddane**
Code cleanup similar to that in https://github.com/zendesk/radar/pull/123.  Also, minor changes as per vanchi's review of the earlier PR.

/cc @zendesk/zendesk-radar

### Steps to merge
 - [ ] :+1: of the team

### References
 - Jira link: https://zendesk.atlassian.net/browse/RADAR-457

### Risks
 - Low: There is the possibility of creating more confusion by the cleanup, but a code review will greatly reduce the risk of this.

### CODE REVIEW

**Comment by vanchi-zendesk** [comment]
:+1:

File: tests/client.presence.sentry.test.js [hunk 1]
```diff
@@ -82,7 +82,10 @@ describe('given a client and a server,', function() {
           assert.ok((ts[2] - ts[1]) < 6000, 'sentry expiry was '+(ts[2] - ts[1]));
           // user expiry = 1000
           assert.ok((ts[3] - ts[2]) >= 900, 'user expiry was '+(ts[3] - ts[2]));
+<<<<<<< HEAD
+=======
           // user expiry = 1000
+>>>>>>> origin/master
```
Comment (vanchi-zendesk) [inline tests/client.presence.sentry.test.js]: This is bad.

File: tests/client.presence.sentry.test.js [hunk 2]
```diff
@@ -122,9 +125,6 @@ describe('given a client and a server,', function() {
             assert.ok((ts[2] - ts[1]) >= 3000, 'sentry expiry was '+(ts[2] - ts[1]));
             // user expiry = 1000
             assert.ok((ts[3] - ts[2]) >= 900, 'user expiry was '+(ts[3] - ts[2]));
-            // user expiry = 1000
-            assert.ok((ts[3] - ts[2]) >= 900, 'user expiry was '+(ts[3] - ts[2]));
```
Comment (vanchi-zendesk) [inline tests/client.presence.sentry.test.js]: And this too.

Comment (vanchi-zendesk) [inline tests/client.presence.sentry.test.js]: Nice catch. Lol. Didn't notice the lines were same.

**Comment by bolddane** [comment]
I will close this PR *without* merging it; the equivalent changes are in PR #125, referenced above.

**Pull request closed without merging**